### ISSUE TITLE
[a11y] missing label for teamfolder name input

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4470]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Have proper label on the input - either with `<label>` or with `aria-label` attribute.

### Actual behaviour
No label at all.

<img width="1984" height="572" alt="Image" src="https://github.com/user-attachments/assets/eff44377-15dd-4046-a8d4-38626506beb1" />
